bin/generate.php

```diff
@@ -58,4 +63,4 @@
 $host = $cmd['host'] ?? null;
 $stream_host = $cmd['stream_host'] ?? null;
 
-(new FileGeneration($dir, $name, $title, $theme, $public_id, $primary_color, $text_color, $background_color, $host, $stream_host))->generate();
+(new FileGeneration($dir, $name, $title, $remote_url, $theme, $public_id, $primary_color, $text_color, $background_color, $host, $stream_host))->generate(true);
```

Comment:
unless otherwise specified/agreed, all new function arguments should go to the end for backwards compatibillity.